src/backend/db/migrations/1_initial-migrations.sql

```diff
@@ -10,8 +10,11 @@ CREATE TABLE "user" (
   image_url TEXT
 );
 
--- table for CM's para list will go here
-
+CREATE TABLE "paras_assigned_to_case_manager" (
+  case_manager_id UUID REFERENCES "user" (user_id) ON DELETE SET NULL,
```

Comment:
these rows are only meaningful if both columns have a value, so I would recommend adding a `NOT NULL` check and dropping the row on relational deletes

Reply:
I've taken this out in the upcoming commit.